tools/lint/json_lint.py

```
@@ -0,0 +1,120 @@
+    if out_of_range:
+        issues = True
+        print(f"❌ {path.name}: out-of-range numeric values detected at {out_of_range[:5]}")
+
+    if schema and jsonschema:
+        try:
+            with schema.open("r", encoding="utf-8") as handle:
+                schema_payload = json.load(handle)
+            jsonschema.validate(instance=payload, schema=schema_payload)
+        except jsonschema.ValidationError as exc:  # pragma: no cover
+            issues = True
+            print(f"❌ {path.name}: schema violation – {exc.message}")
```

Ruff (0.14.1)

String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)